hack/get-build-ld-flags.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+#!/usr/bin/env -S bash -e
```

Comment:
What is `-S` flag about?

```bash
$ /usr/bin/env -S bash -e
/usr/bin/env: invalid option -- 'S'
Try '/usr/bin/env --help' for more information.
```

```bash
$ /usr/bin/env --version
env (GNU coreutils) 8.28
Copyright (C) 2017 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by Richard Mlynarik and David MacKenzie.
```

Reply:
That seems to be an option in newer env versions. I used a more compatible version now.